### PR TITLE
8 emoji for 17

### PR DESCRIPTION
**Pull request by eggrobin**
From ESC.

[[181-C5](https://www.unicode.org/cgi-bin/GetL2Ref.pl?181-C5)] Consensus: Accept 8 new emoji characters with the following code points and character names, based on Section 2 of document [L2/24-226R](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/24-226R), for Unicode Version 17.0:

        U+1FAEA DISTORTED FACE
        U+1FAEF FIGHT CLOUD
        U+1FADD APPLE CORE
        U+1FACD ORCA
        U+1FAC8 HAIRY CREATURE
        U+1FA8A TROMBONE
        U+1F6D8 LANDSLIDE
        U+1FA8E TREASURE CHEST

* https://github.com/unicode-org/utc-release-management/issues/172

### CODE REVIEW

**Review by nedley — DISMISSED** [review]

File: unicodetools/data/ucd/dev/EastAsianWidth.txt [hunk 1]
```diff
@@ -2624,6 +2624,7 @@ FFFD           ; A  # So         REPLACEMENT CHARACTER
 1F6D0..1F6D2   ; W  # So     [3] PLACE OF WORSHIP..SHOPPING TROLLEY
 1F6D3..1F6D4   ; N  # So     [2] STUPA..PAGODA
 1F6D5..1F6D7   ; W  # So     [3] HINDU TEMPLE..ELEVATOR
+1F6D8          ; N  # So         LANDSLIDE
```
Comment (markusicu) [inline unicodetools/data/ucd/dev/EastAsianWidth.txt:2668]: Do we know why some of these are W vs. N?

**Review by markusicu — DISMISSED** [review]

File: unicodetools/data/ucd/dev/EastAsianWidth.txt (hunk 1 above)
Comment (eggrobin) [reply unicodetools/data/ucd/dev/EastAsianWidth.txt:2668]: Oh. They should probably be W, like all recent emoji. https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5Cp%7BEmoji%7D&g=age+ea&i=

Comment (markusicu) [reply unicodetools/data/ucd/dev/EastAsianWidth.txt]: Interesting... especially weird that the "User interface symbols" from Unicode 6 are inconsistent with each other.

**Review by markusicu — APPROVED** [review]

File: unicodetools/data/ucd/dev/EastAsianWidth.txt (hunk 1 above)
Comment (nedley) [reply unicodetools/data/ucd/dev/EastAsianWidth.txt]: At the risk of summoning an historian to find out why, U+231A and U+231B were changed from eaw=N to eaw=W in Unicode 9.0.

**Comment by nedley** [comment]
Should we have an invariant checking East_Asian_Width and Vertical_Orientation for Emoji_Presentation, at least for assignments from now on?

**Comment by markusicu** [comment]
> Should we have an invariant checking East_Asian_Width and Vertical_Orientation for Emoji_Presentation characters, at least for assignments from now on?

Is there a good correlation? Or is it more like "recently added emoji have been assigned eaw=W"?
If there is a good correlation, I suggest you create a PAG issue and propose certain property values going forward.
And maybe look for old emoji that use different values for seemingly no good reason.

**Comment by nedley** [comment]
Filed https://github.com/unicode-org/properties/issues/347 to get the ball rolling. Thanks!

**Comment by eggrobin** [comment]
I think we forgot a property for these characters, namely the Emoji property 🙃

**Pull request merged**